catalog/transformer/catalog-transformer-video-input/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -13,7 +13,13 @@
  **/
  -->
 <blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0">
-    <bean id="transformer" class="ddf.catalog.transformer.input.video.VideoInputTransformer"/>
+    <bean id="transformer" class="ddf.catalog.transformer.input.video.VideoInputTransformer">
+        <argument ref="genericMetacardType"/>
+    </bean>
+
+    <reference id="genericMetacardType" interface="ddf.catalog.data.MetacardType"
```

Comment:
Yes - the definition of the metacardtype needs to be pulled out and made common.  I'm thinking this "base" metacardtype should be moved to core.